javascript/javascript3/week1/Giphy/index.js

```diff
@@ -0,0 +1,52 @@
+const userInput = document.querySelector(".gifName");
+let gifArray;
+document.querySelector(".search").addEventListener("click", getGif);
+async function getGif() {
+  if (userInput.value.trim() === "") {
+    alert("You forgot to enter giphy-name");
+  } else if (amount.value === "") {
+    giphys = `https://api.giphy.com/v1/gifs/search?api_key=Y55ch22OYmhAWV15dqMxf0PDOtgBzrlL&q=${userInput.value}&limit=10&offset=0&rating=g&lang=en`;
```

Comment:
you don't need need to do this checking if you specify the limit by default as 10. 
```
async function getGif() {
    if (userInput.value.trim() === "") {
        alert("You forgot to enter giphy-name");
    }
    searchTerm = userInput.value
    // Fetch and respose data from API
    const response = await fetch(apiUrl + userInput.value + `&limit=${limit}&offset=0&rating=g&lang=en`);
    const info = await response.json();
    gifArray = info.data;
    /*  remove();
     const ul = document.createElement("ul");
     const div = document.querySelector(".showGifs");
     div.appendChild(ul);
     renderGifs(); */
}
```
your function will be some thing like this